src/widgets/settingspages/FiltersPage.cpp

```diff
@@ -0,0 +1,112 @@
+#include "FiltersPage.hpp"
+
+#include "controllers/filters/FilterModel.hpp"
+#include "singletons/Settings.hpp"
+#include "util/LayoutCreator.hpp"
+#include "widgets/dialogs/ChannelFilterEditorDialog.hpp"
+#include "widgets/helper/EditableModelView.hpp"
+
+#include <QTableView>
+
+#define FILTERS_DOCUMENTATION \
```

Comment:
the documentation is good, but should before merging, move to the wiki instead. https://github.com/Chatterino/wiki